### PR TITLE
Fix updating error of ress with moduleDetection=false

### PR DESCRIPTION
**Pull request by kachick**
- **`npm uninstall ress`**
- **Replace ress with modress**

Closes #1617

I think this is relaetd to following issues

- https://github.com/tsconfig/bases/pull/284
- https://github.com/filipelinhares/ress/pull/17

### CODE REVIEW

**Comment by kachick** [comment]
Before
---

image

No Reset
---

image

After
---

image

File: index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
       name="description"
       content="A web application to manage participants and timer for mob-programming!"
     />
+    <link rel="stylesheet" href="https://unpkg.com/ress/dist/ress.min.css">
```
Comment (kachick) [inline index.html:15]: https://github.com/filipelinhares/ress/blob/7092c6033e6db09d8e40a161831e86071012be52/README.md?plain=1#L60-L61

Drop to depend npm and using unpkg looks simple solution to me.

File: src/index.tsx [hunk 2]
```diff
@@ -2,7 +2,6 @@ import { StrictMode } from "react";
 import { createRoot } from "react-dom/client";
 import App from "./App";
 
-import "ress";
```
Comment (kachick) [inline src/index.tsx:5]: Introduced in https://github.com/mobu-of-the-world/mobu/commit/d65888447745dba8951786c04e0386485e4db73c#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R19

@pankona

**Comment by pankona** [comment]
Thanks! Looks good solution for me too 👍

**Comment by ujihisa** [comment]
:goodpoem:

**Pull request merged**